### PR TITLE
Update read me arks

### PR DESCRIPTION
**Pull request by MurathanGoktas**
ReadMe update for ARCS after ARKS integration.

### CODE REVIEW

**Comment by warrenlr** [comment]
could we please also update the demos?

**Comment by lcoombe** [comment]
> could we please also update the demos?

@warrenlr - Murathan added in the ARKS demo to ARCS in a previous PR, and changed it to use the new arks-make Makefile in this repo. Any other changes that you had in mind?

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ If your boost library headers are not in your PATH you can specify their locatio
 
 ### ARCS+LINKS Pipeline
```
[the remark this answers is not in the document]
Comment (MurathanGoktas) [reply README.md]: I think explanation in this section holds for both methods that's why I didn't mention 'default' in the header or inside the section. Should I?

[the remark this answers is not in the document]
Comment (lcoombe) [reply README.md]: Yes good point. Ok then leave this part as is, then you could add some information below on specifically how to run ARCS (with read alignments) vs. ARCS (arks mode)

File: README.md [hunk 2]
```diff
@@ -48,8 +49,34 @@ When using the `-D`/`--dist_est` ARCS option to estimate gap sizes, the user is
...
+
+ARKS is another scaffolding tool developed by our group designed based on kmerization paradigm. ARKS is integrated into ARCS(v1.x.x) for better usability.
+
+For running the pipeline with ARKS, Makefile should target `arks` with `./arcs-make arks`. For more info check `./arcs-make help`.
+
+To run `./arcs` executable with ARKS, run `./arcs --arks`. For more info check `./arcs --help`.
+
+### Demo
+
 you can test your installation by following instructions at: Examples/arcs_test-demo/README.txt
```
Comment (lcoombe) [inline README.md]:
```suggestion
You can test your installation by running one of our supplied demos:
* ARCS: `Examples/arcs_test-demo`
* ARKS: `Examples/arks_test-demo`
For both, you can compare your output to the files provided in the `output` folders within the above directories.
```

File: README.md [hunk 3]
```diff
@@ -48,8 +49,34 @@ When using the `-D`/`--dist_est` ARCS option to estimate gap sizes, the user is
...
+for ARKS: Examples/arks_test-demo/README.txt
```
Comment (lcoombe) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -48,8 +49,34 @@ When using the `-D`/`--dist_est` ARCS option to estimate gap sizes, the user is
...
+
```
Comment (lcoombe) [inline README.md]: You can remove the rest of the demo text below here -- I think it is redundant with my suggestion above.

File: README.md [hunk 5]
```diff
@@ -48,8 +49,34 @@ When using the `-D`/`--dist_est` ARCS option to estimate gap sizes, the user is
...
+Default mode used read alignments to contigs.
+
+For running the pipeline default mode, Makefile should target `arcs` with `./arcs-make arcs`. For more info check `./arcs-make help`.
+
+To run `./arcs` executable in default mode, run `./arcs` without additional flag. For argument requiriments check `./arcs --help`.
+
+### Running ARCS in '--arks' mode
+
+ARKS is another scaffolding tool developed by our group designed based on kmerization paradigm. ARKS is integrated into ARCS(v1.x.x) for better usability.
```
[the remark this answers is not in the document]
Comment (lcoombe) [reply README.md]: Hey Murathan -- did this text get lost in the shuffle somewhere?

File: README.md [hunk 6]
```diff
@@ -48,8 +54,29 @@ When using the `-D`/`--dist_est` ARCS option to estimate gap sizes, the user is
...
+
+To run the pipeline in default mode, run `Examples/arcs-make arcs`. For more info check `Examples/arcs-make help`.
+
+To run the `arcs` executable in default mode, run `arcs <alignments>`. For descriptions of all arguments, run `arcs --help`.
+
+### Running ARCS in '--arks' mode
+
+To run the pipeline in ARKS mode, run `Examples/arcs-make arks`. For more info check `Examples/arcs-make help`.
+
+To run the `./arcs` executable in ARKS mode, run `./arcs --arks`. For descriptions of all arguments, run `./arcs --help`.
```
Comment (lcoombe) [inline README.md]:
```suggestion
To run the `arcs` executable in ARKS mode, run `arcs --arks`. For descriptions of all arguments, run `arcs --help`.
```
Forgot to change this one too in my last round of suggestions to keep it consistent with ARCS

File: README.md [hunk 7]
```diff
@@ -48,8 +54,29 @@ When using the `-D`/`--dist_est` ARCS option to estimate gap sizes, the user is
...
+
+You can test your installation by running one of our supplied demos:
```
Comment (lcoombe) [inline README.md]: There was a 'Demo' header here that got deleted accidentally I think.

Comment (MurathanGoktas) [reply README.md]: Oh I thought you wanted me to delete it. Putting it again

File: README.md [hunk 8]
```diff
@@ -2,13 +2,19 @@
 
 # ARCS
 
-Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data.
+Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data. 
+ARCS can be run in 2 modes:
+* [ARCS](https://doi.org/10.1101/100750) (default) uses alignments of linked reads to the input contigs 
+* [ARKS](https://doi.org/10.1186/s12859-018-2243-x) (`--arks`) uses exact k-mer mapping associate linked reads to input contigs
```
Comment (lcoombe) [inline README.md]:
```suggestion
* [ARKS](https://doi.org/10.1186/s12859-018-2243-x) (`--arks`) uses exact k-mer mapping to associate linked reads to input contigs
```

**Comment by lcoombe** [comment]
Thanks Murathan! :1st_place_medal:

**Pull request merged**